libplatsupport/src/plat/polarfire/mstimer.c

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright 2020, DornerWorks
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <stdio.h>
+#include <assert.h>
+#include <errno.h>
+#include <utils/util.h>
+#include <platsupport/timer.h>
+#include <platsupport/plat/mstimer.h>
+#include <utils/frequency.h>
+
+#include "../../ltimer.h"
+
+#define MAX_TIMEOUT_NS (BIT(31) * (NS_IN_S / POLARFIRE_PCLK))
+
+static void mstimer_print_regs(mstimer_t *mstimer)
```

Comment:
```suggestion
static inline void mstimer_print_regs(mstimer_t *mstimer)
```
Cold add `inline`, so there is no compiler warning in case this is not used. Is this used anywhere at the moment?

Reply:
Done. It is not although I do not see a compiler warning.